python_bindings/src/halide/__init__.py

```diff
@@ -22,3 +22,4 @@
 
 from halide_ import *
 from halide_ import _, _1, _2, _3, _4, _5, _6, _7, _8, _9
+from _generator_helpers import *
```

Comment:
Do we need this here? This line dumps everything from `_generator_helpers` into our public API. Is that intentional? It makes sense for `halide_`, which is designed to be outward-facing.

Reply:
It should probably be `from ._generator_helpers` too (note the `.`).

Reply:
We need some of it. E.g. `halide.GeneratorParam` which is defined in _generator_helpers but we want as part of our public API. Let me see if I can selectively import instead.

Reply:
Using `from ._generator_helpers` didn't work for me -- stuff didn't get imported

Reply:
After merge with latest stuff, the `from ._generator_helpers` works properly now